omnia/config.sh

```diff
@@ -181,6 +183,20 @@ importOptionsEnv () {
 	[[ ${errors[*]} ]] && { printf '%s\n' "${errors[@]}"; exit 1; }
 }
 
+importServicesEnv () {
+	local _config="$1"
+	local _json
+
+	_json=$(jq -S '.services' < "$_config")
+	#extract cmc key
+	CMC_API_KEY="${CMC_API_KEY-$(jq -r '.cmcApiKey' <<<"$_json")}"
```

Comment:
You can remove the `_json` var and read the desired value directly from the config:

```suggestion
	CMC_API_KEY="${CMC_API_KEY-$(jq -r '.services.cmcApiKey' "$_config")}"
```